build_tools/benchmarks/export_benchmark_config.py

```diff
@@ -52,7 +52,10 @@
         "x86_64",
     "cuda":
         lambda config: config.target_device_spec.architecture.architecture ==
-        "cuda",
+        "cuda" and "long-running" not in config.module_execution_config.tags,
+    "cuda-long":
+        lambda config: config.target_device_spec.architecture.architecture ==
+        "cuda" and "long-running" in config.module_execution_config.tags,
```

Comment:
Instead of filtering on the tag of execution config, maybe we can simply filter on the model's tag?
```suggestion
    "cuda-long":
        lambda config: config.target_device_spec.architecture.architecture ==
        "cuda" and "long-running" in config.module_generation_config.imported_model.model.tags,
```

Reply:
We may end up using the same model to run under different scenarios so would prefer to not have a 1:1 relationship between a model and how it is executed.

Reply:
To me it sounds like the characteristic of "long benchmark" is the combination of model + execution config, which is represented by `E2EModelRunConfig`:
https://github.com/openxla/iree/blob/4e99afbe7a516e6437f1f4505e3544900e50749f/build_tools/python/e2e_test_framework/definitions/iree_definitions.py#L297-L310

So maybe adding a new "tags" field to it is a better way? Another problem to have "long execution config" is that if we want to move benchmarks between "long" and "non-long" groups, we will change their execution config, which will generate different benchmark series.

Reply:
I see, thanks for pointing that out. I'll add a tags field to E2EModelRunConfig.

Reply:
Updated to use tags in E2EModelRunConfig. PTAL